src/Microsoft.Azure.WebJobs.Extensions.Storage/TimeoutHandler.cs

```diff
@@ -2,21 +2,27 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.Diagnostics;
 using System.Runtime.ExceptionServices;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
 using Microsoft.Azure.WebJobs.Host.Timers;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.WebJobs.Extensions.Storage
 {
     internal static class TimeoutHandler
     {
-        private static readonly TimeSpan DefaultTimeout = TimeSpan.FromMinutes(2);
+        private static readonly TimeSpan DefaultTimeout = TimeSpan.FromMinutes(3);
 
-        public static async Task<T> ExecuteWithTimeout<T>(string operationName, string clientRequestId, IWebJobsExceptionHandler exceptionHandler, Func<Task<T>> operation)
+        public static async Task<T> ExecuteWithTimeout<T>(string operationName, string clientRequestId,
+            IWebJobsExceptionHandler exceptionHandler, ILogger logger, CancellationToken cancellationToken, Func<Task<T>> operation)
```

Comment:
This is kind of clunky, but this `cancellationToken` isn't used to cancel anything. But in order to determine if it was a network timeout (for logging only), I need to know if someone canceled the operation directly. Since this is hopefully a short-lived API, I'm okay with it being odd for now and this whole class will go away in a future update.